### PR TITLE
Update docker image, finish support, add setup

### PR DESCRIPTION
**Pull request by Linux123123**

### CODE REVIEW

File: scripts/troubleshooting.sh [hunk 1]
```diff
@@ -39,21 +62,49 @@ detect_distro() {
 
 panel_logs(){
   if [ -f "/var/www/pterodactyl/storage/logs/laravel-$(date +%F).log" ]; then 
-    panel_log="$(tail -n 100 /var/www/pterodactyl/storage/logs/laravel-"$(date +%F)".log | nc bin.ptdl.co 99)"
+    panel_log="$(nc termbin.com 9999 < /var/www/pterodactyl/storage/logs/laravel-"$(date +%F)".log | tr -d '\0')"
```
Comment (vilhelmprytz) [inline scripts/troubleshooting.sh]: Would it be feasible to use something else than termbin? Would it also be possible to not hardcode it?

File: scripts/troubleshooting.sh [hunk 2]
```diff
@@ -39,21 +62,49 @@ detect_distro() {
...
   else
     panel_log='Empty'
   fi
 }
 
+wings_logs(){
+  if [ -f "/var/log/pterodactyl/wings.log" ]; then
+    wings_log="$(nc termbin.com 9999 < /var/log/pterodactyl/wings.log | tr -d '\0')"
```
Comment (vilhelmprytz) [inline scripts/troubleshooting.sh]: Linked to my previous comment :slightly_smiling_face:

File: src/modules/Functions.ts [hunk 3]
```diff
@@ -1,20 +1,28 @@
-import { Guild, MessageEmbed, TextChannel } from "discord.js";
+import {
+  Guild,
+  MessageAttachment,
+  MessageEmbed,
+  PermissionOverwrites,
```
Comment (vilhelmprytz) [inline src/modules/Functions.ts]:
```suggestion
  MessageEmbed,
```

`PermissionOverwrites` is defined but not used.

**Review by vilhelmprytz — COMMENTED** [review]
Looks good. But then I don't know what I am doing. I did leave some comments, however.

File: scripts/troubleshooting.sh (hunk 1 above)
Comment (Linux123123) [reply scripts/troubleshooting.sh]: Sure, but do you mean like ENV var or just have a variable in the script?

Comment (vilhelmprytz) [reply scripts/troubleshooting.sh]: An environment variable that would be specified in `.env` (which is read by `docker-compose`). I presume, without reading the code too closely, that the bot webserver actually reads the `.sh` file so you could probably fill in variables at that step?

Comment (Linux123123) [reply scripts/troubleshooting.sh]: When I load the script I can just replace the URL with the one in the ENV var. It is pretty straight forward.

Comment (Linux123123) [reply scripts/troubleshooting.sh]: Fixed

File: scripts/troubleshooting.sh (hunk 2 above)
Comment (Linux123123) [reply scripts/troubleshooting.sh]: Fixed

**Comment by Linux123123** [comment]
LGTM

**Pull request merged**